docs/manual/docs/cmd/teams/report/report-deviceusageuserdetail.md

```diff
@@ -15,6 +15,8 @@ Option|Description
 `--help`|output usage information
 `-p, --period [period]`|The length of time over which the report is aggregated. Supported values `D7|D30|D90|D180`
 `-d, --date [date]`|The date for which you would like to view the users who performed any activity. Supported date format is `YYYY-MM-DD`.
+`-f, --outputFilePath [outputFilePath]`|Path to the file where the upgrade report should be stored in
```

Comment:
This command doesn't generate an upgrade report. Let's reword it to reflect the contents of the report

Reply:
Done. This is fixed